### PR TITLE
Fix time zones difference in tables

### PR DESCRIPTION
**Pull request by xdanielsb**

### CODE REVIEW

File: tests/unit/utils.spec.js [hunk 1]
```diff
@@ -53,3 +53,17 @@ describe('Test toggle element from list', () => {
     }
   );
 });
+
+describe('Test convert date to UTC', () => {
+  const date = new Date('Wed, 15 Jun 2020 07:00:00 GMT');
```
Comment (antoinerabany) [inline tests/unit/utils.spec.js:58]: Create a date as Europe/Paris and convert it to UTC and verify you have the wanted result.

File: src/components/CMessageRow.vue [hunk 2]
```diff
@@ -99,15 +99,17 @@ export default {
         return null;
       }
       const factor = (1 - this.progress) / this.progress;
-      const diff = utils.formatMillis((new Date() - this.startedDatetime) * factor);
+      const diff = utils.formatMillis(
```
Comment (antoinerabany) [inline src/components/CMessageRow.vue:102]: what is the timezone of this.startedDatetime here ? you cannot compare 2 dates as you do with different timezones.

Comment (xdanielsb) [reply src/components/CMessageRow.vue:102]: umm I think it is UTC,  [started date timezone](https://github.com/wiremind/remoulade/blob/master/remoulade/state/middleware.py), I just change to UTC when is instanced new Date()

File: tests/unit/utils.spec.js (hunk 1 above)
Comment (xdanielsb) [reply tests/unit/utils.spec.js:58]: I think I have that in the test?, I am creating a date with timeZone 'Europe/Paris', convert it to UTC and check if the difference is 2 hours, however, I am not sure if the implementation is right.

**Pull request merged**